uint8 x_array
uint8 y_array
uint8 BUTTON_UP=0
uint8 BUTTON_DOWN=1
uint8 buttom_a
uint8 buttom_b
uint8 buttom_c